fineract-core/src/main/java/org/apache/fineract/infrastructure/core/service/database/DatabaseSpecificSQLGenerator.java

```diff
@@ -289,4 +289,13 @@ public Long fetchPK(GeneratedKeyHolder keyHolder) {
             }
         };
     }
+
+    public String incrementDateByOne(String date){
```

Comment:
I think the method name should be `incrementDateByOneDay` and the parameter name should be dateColumn or something.